Add docs for Canny (Actions) Destination (#4948)
segmentio/segment-docs: src/connections/destinations/catalog/canny-functions/index.md

---
title: Canny (Actions) Destination

> id: <integration_id>

Author: I'm not sure what to write for id. I was not able to find the id of our destination in private beta.

> id: <integration_id>

Reviewer: ```suggestion
id: 6489bbade6fe3eb57c13bd6a
beta: true
hidden: true
private: true
```

---

{% include content/plan-grid.md name="actions" %}

> [Canny](https://canny.io?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a single place for all customer feedback. It saves you time managing all the feedback while keeping your customers in the loop. Let your customers post and vote on feedback from within your website or mobile app. You'll get an organized list of feedback that you can use to inform your roadmap.

Reviewer: ```suggestion
[Canny](https://canny.io?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a single place for all customer feedback. It saves you time managing all the feedback while keeping your customers in the loop. Let your customers post and vote on feedback from within your website or mobile app. You'll get an organized list of feedback that you can use to inform your roadmap.
```

> This destination is maintained by Canny. For any issues with the destination, [contact the Canny Support team](mailto:segment-help@canny.io).

Reviewer: ```suggestion
Canny maintains this destination. For any issues with the destination, [contact the Canny Support team](mailto:segment-help@canny.io).
```

> 

Reviewer: ```suggestion

> success ""
> **Good to know**: This page is about the [Actions-framework](/docs/connections/destinations/actions/) Canny Segment destination. There's also a page about the [non-Actions Canny destination](/docs/connections/destinations/catalog/canny/). Both of these destinations receive data from Segment.
```

{% include content/ajs-upgrade.md %}

## Getting started

> 1. Go to your [Canny Admin Segment Settings](https://canny.io/redirect?to=%2Fadmin%2Fsettings%2Fsegment).

Reviewer: ```suggestion
1. Go to your [Canny Admin Segment Settings](https://canny.io/redirect?to=%2Fadmin%2Fsettings%2Fsegment){:target="_blank"}.
```

2. Obtain your API key by installing the integration.

> 3. From the Segment web app, click **Catalog**, then click **Destinations**.

Reviewer: ```suggestion
3. From the Segment web app, navigate to **Connections > Catalog**, then select the **Destinations** tab in the catalog.
```

> 4. Find the Destinations Actions item in the left navigation, and click it.

Reviewer: ```suggestion
4. Search for *Canny (Actions)* and select it.
```

5. Click **Configure Canny (Actions)**.
6. Select an existing Source to connect to Canny (Actions).

> 7. After you finished configuring the destination, update the API key in the Canny (Actions) destination settings, enable destination and save changes.

Reviewer: ```suggestion
7. Enter your configuration settings. 
8. Update the API key in the Canny (Actions) destination settings.
9. Enable the destination and save changes.
```

{% include components/actions-fields.html %}

## Identify

If you haven't had a chance to review spec, take a look to understand what the Identify method does. An example call would look like:

```js
analytics.identify("userId123", {
  email: "john.doe@example.com",
  name: "John Doe",
});
```

Identify calls will be sent to Canny as an identify event. Once identified, users will appear in Canny and will appear in your vote-on-behalf feature. The `name` and `email` traits are **required** fields.

## Group

An example call would look like:

```js
analytics.group("0e8c78ea9d97a7b8185e8632", {
  name: "X Corp",
});
```

Group calls will be sent to Canny as an group event. Once sent, groups will appear in Canny as companies attached to users. The `name` trait is a **required** field.

### Custom Fields

> Custom Fields is a list of [traits](/docs/connections/spec/identify/#traits) to be imported as custom fields for Identify events.

Reviewer: @BR4VE are you sure you want to include these sections? These sections aren't needed for actions destinations and are usually only needed for the classic docs.

Author: @stayseesong Thanks for mentioning this. I was not sure it it's required or not. Now I removed it 👍
